docs/project/contribution_tools.md

```diff
@@ -351,6 +352,23 @@ lldb bazel-bin/toolchain/carbon
 Any installed version of LLDB at least as recent as the installed Clang used for
 building should work.
 
+### Using LLDB from the command line
+
+We include launch commands for running lldb in VSCode in
+[`.vscode/lldb_launch.json`](/.vscode/lldb_launch.json). But it's also possible
+to run lldb from the command line.
+
+When running the debugger, include the `--local-lldbinit` argument to use our
+preset configuration options.
```

Comment:
Is it worth noting that running from subdirectories won't work? I believe that's implied for `.lldbinit`, but might still be worth specifying (I admit I'm surprised there doesn't seem to be a way to pass in an arbitrary path to a `.lldbinit`)

```suggestion
preset configuration options. This requires running from the repository root.
```

Reply:
The use of `project_root` may have fixed this.. nope never mind. Then the lldbinit file is not used. Done